### PR TITLE
Use OTLP Metrics type instead of OpenCensus type [aws-ecs-container-metrics-receiver]

### PR DESCRIPTION
**Pull request by hossain-rayhan**
**Description:** 
Use `pdata.Metrics` instead of OpenCensus metric types. 
This change replaces all the uses of OpenCensus metric types with OTLP metric type (`pdata.Metrics`) and updates the unit tests.

**Link to tracking Issue:** 
#1122 

**Testing:** 
Updated previous unit test.
Manually tested with `logging` exporter for e2e experience.

### CODE REVIEW

File: receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/resource.go [hunk 1]
```diff
@@ -17,35 +17,38 @@ package awsecscontainermetrics
...
+	attributes[AttributeECSTaskRevesion] = pdata.NewAttributeValueString(tm.Revision)
+	attributes[AttributeECSServiceName] = pdata.NewAttributeValueString("undefined")
+
+	return createResourceWithAttributes(attributes)
+}
+
+func createResourceWithAttributes(attributes map[string]pdata.AttributeValue) pdata.Resource {
+	resource := pdata.NewResource()
+	resource.InitEmpty()
+	resource.Attributes().InitFromMap(attributes)
```
Comment (anuraaga) [inline receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/resource.go:50]: Not a big deal, but mutating the attributes directly would avoid the overhead of an extra map allocation

**Review by anuraaga — APPROVED** [review]
Great to see migration, thanks!

File: receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/resource.go (hunk 1 above)
Comment (hossain-rayhan) [reply receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/resource.go]: Thanks. Pushed an update.

**Comment by hossain-rayhan** [comment]
@tigrannajaryan @bogdandrutu Would appreciate a look to see if we can merge it.

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**